### DISCUSSION TITLE
Cannot install Deta CLI

### DISCUSSION THREAD

**Discussion by bistiks1979** [deta/cloud-docs#692] [Help]
Cannot install Deta CLI please help

**Comment by sudo-subg** [comment]
Will need more than this to help you. Like what problem are you facing, which OS and all. Make sure you are following instructions provided [here](https://docs.deta.sh/docs/cli/install).

**Reply by bistiks1979** [reply]
i found the solution thanks wrongs commands but i have another problem now i create everything good and works but after i close internet my micro  and adress dissapear how i bring back micro in my account ?

**Reply by bistiks1979** [reply]
endpoint  adress says 404 not found ho i open my control panel again
?
